loadgen-terraform/env_config.txt

```diff
@@ -0,0 +1,4 @@
+RABBIT_PASS=
```

Comment:
All blanks? Why?

Reply:
Didn't want to put ips and passwords in a public repo, I know they're only for the wargames env but I felt it was safer to not.